core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java

```diff
@@ -932,7 +944,6 @@ private void parsePersistence(final ConfigurationReader reader, final Configurat
                ignoreElement(reader, element);
                break;
             case SINGLE_FILE_STORE:
-               CONFIG.warnUsingDeprecatedClusterLoader();
```

Comment:
Whoops, this was a mistake. But we should still probably have a warning about the single file store or no? I would guess we want to push people to use the regular file store (SIFS) instead?